dataset_preprocessing/tutorials/CONUS404/conus404_explore.ipynb

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
is this notebook an update to this one? https://github.com/hytest-org/hytest/blob/main/dataset_access/conus404_explore.ipynb

Reply:
If so, it should probably go in `dataset_access/tutorials` (showing how to view and access a dataset, rather than how to process it)

Reply:
@amsnyder, thats merely my reference working book and will be deleted upon completion of the preprocessing notebook.